backstage/plugins/knative-event-mesh-backend/src/providers/knativeEventMeshProvider.ts

```diff
@@ -110,7 +116,7 @@ export class KnativeEventMeshProvider implements EntityProvider {
     constructor(config:KnativeEventMeshProviderConfig, logger:Logger, taskRunner:TaskRunner) {
         this.env = config.id;
         this.baseUrl = config.baseUrl;
-
+        this.token = config.token;
```

Comment:
Can you make sure that the token exists? Validate it possibly in `static fromConfig` method?

Reply:
This is done now